packages/transaction-controller/CHANGELOG.md

```diff
@@ -5,8 +5,55 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [17.0.0]
+### Added
+- **BREAKING:** Add swaps logic into `TransactionController` ([#1877](https://github.com/MetaMask/core/pull/1877))
+  - Swap transaction updates can be prevented by setting `disableSwaps` as `true`. If not set it will default to `false`.
+  - This value MUST be set to true in mobile, since mobile doesn't use any swap logic.
```

Comment:
I think we should omit this to avoid coupling the changelog to specific clients.

Reply:
```suggestion
```